site/docs/master/velero-install.md

```diff
@@ -0,0 +1,46 @@
+# Velero Install CLI
+
+This document serves as a guide to using the `velero install` CLI command to install `velero` server components into your kubernetes cluster.
+
+_NOTE_: `velero install` will, by default, use the version information to determine the version of of the server compoents to deploy. This behavior may be overridden by using the `--image` flag. Refer to [Building Server Component Container Images For Server Component][1].
```

Comment:
`of of` -> `of`

Maybe simplify the link text to `Building Server Component Container Images`